src/python/gudhi/wasserstein/wasserstein.py

```diff
@@ -173,9 +173,40 @@ def wasserstein_distance(X, Y, matching=False, order=2., internal_p=2., enable_a
         return ep.concatenate(dists).norms.lp(order).raw
         # We can also concatenate the 3 vectors to compute just one norm.
 
-    # Comptuation of the otcost using the ot.emd2 library.
+    # Comptuation of the otcost using the ot.emd2 library, and matching/transport plan using ot.emd.
     # Note: it is the Wasserstein distance to the power q.
     # The default numItermax=100000 is not sufficient for some examples with 5000 points, what is a good value?
```

Comment:
This comment needs to be updated as default numItermax=2000000

Reply:
IIRC, numItermax=100000 is the default in POT, we override it with a larger number, and this comment explains why.